server/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const isString = (obj: any): obj is string =>
+  Object.prototype.toString.call(obj) === "[object String]";
```

Comment:
https://lodash.com/docs/#isString

Reply:
This is a typescript typeguard notation.

See -> obj is string =>

Reply:
Yes, but lodash has built-in method to check it.

Reply:
This is not the same. This is type guard. lodash is missing obj is string part.